encord/client.py

```diff
@@ -118,6 +120,9 @@
 from encord.utilities.client_utilities import optional_set_to_list, parse_datetime
 from encord.utilities.project_user import ProjectUser, ProjectUserRole
 
+LONG_POLLING_RESPONSE_RETRY_N = 3
+LONG_POLLING_SLEEP_ON_FAILURE_SECONDS = 3
```

Comment:
An exponential back off would be what usually happens but this is probably fine.

Reply:
yep, I agree